tests/lz4

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# - compression
+# - iterations
+
+. $LKP_SRC/lib/upload.sh
+. $LKP_SRC/lib/unit.sh
+. $LKP_SRC/lib/reproduce-log.sh
+
+cd $BENCHMARK_ROOT/lz4 || die "no $BENCHMARK_ROOT/lz4"
+
+TESTFILE=ubuntu-18.04.3-desktop-amd64.iso
+if [ ! -f "$TESTFILE" ]; then
+    wget http://old-releases.ubuntu.com/releases/18.04.3/$TESTFILE
+fi
+
+echo "Running lz4 benchmark..."
+
+for i in $(seq $iterations)
+do
+    echo Iterations: $i
+    if [ "$compression" = "1" ]
+    then
+      log_cmd ./lz4 -q -i10 -b1 -e1 $TESTFILE 2>&1
+    fi
```

Comment:
not sure whether these lines can be written as log_cmd ./lz4 -q -i10 -b$compression -e$compression $TESTFILE 2>&1